app/components/DocumentExplorerNode.tsx

```diff
@@ -7,49 +7,40 @@ import breakpoint from "styled-components-breakpoint";
 import Flex from "~/components/Flex";
 import Disclosure from "~/components/Sidebar/components/Disclosure";
 import Text from "~/components/Text";
-import { ancestors } from "~/utils/tree";
 
 type Props = {
-  location: any;
   selected: boolean;
   active: boolean;
   style: React.CSSProperties;
-  isSearchResult: boolean;
   expanded: boolean;
   icon?: React.ReactNode;
+  title: string;
+  nestLevel: number;
```

Comment:
Keep this as `depth`?

Reply:
Yes! Now that that component has `Node` in its name, `depth` would be acceptable.